types/parse/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for parse 2.2.9
+// Type definitions for parse 2.9
```

Comment:
Since your change fixes the lint error, could you remove the "dt-header" rule from the lint file? If you want to you can remove the others too and see what breaks - ideally all of them should be removed.

Reply:
Removed the `dt-header` rule 👍 
Removing the others still results in many errors, but we're thinking of doing another PR soon where we could do a sweep of those.